FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Confirm.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using AutoFixture;
+using FrontDesk.Core.Aggregates;
+using FrontDesk.Core.Events;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Confirm
+  {
+    private Fixture _fixture = new Fixture();
+
+    [Fact]
+    public void ReconfirmResultDateNotChanged()
+    {
+      var appointment = _fixture.Build<Appointment>()
+        .Without(z => z.Events)
+        .Create();
+
+      var confirmedDate = DateTime.Now;
```

Comment:
Avoid using DateTime.Now in tests since it's always different. Use a hard-coded test datetime (ideally a class constant).